### ISSUE TITLE
Implement bindings to the OpenGL library

### ISSUE THREAD

**Issue by ErichDonGubler** [rjs-team/rjs#5]

**Comment by ricochet1k** [comment]
This issue should be broken down into many pieces, the WebGL spec is big. At least WebGL1 vs WebGL2 should be separated.

**Comment by ErichDonGubler** [comment]
When you say the "spec", I'm assuming you're talking about the WebGL API and its versions? We're not actually implementing WebGL, right?

/me has a lot to learn about these technologies

**Comment by ricochet1k** [comment]
Yeah. For the most part they just needs to bind to the corresponding OpenGL functions, in a few cases it requires a little more work.

**Comment by ricochet1k** [comment]
Update: A small part of WebGL 1 has been implemented, enough to render a [simple example](https://github.com/rjs-team/rjs/blob/master/examples/webgl/webgl.js).

**Comment by WolfieWerewolf** [comment]
I have to agree with @ricochet1k ... implementing the two specs backing onto OpenGL is a good objective.  At least from my perspective this is very useful.
